contracts/dev/ocr2dr/OCR2DRClient.sol

```diff
@@ -0,0 +1,129 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.6;
+
+import "./OCR2DR.sol";
+import "./interfaces/OCR2DRClientInterface.sol";
+import "./interfaces/OCR2DROracleInterface.sol";
+
+/**
+ * @title The OCR2DR client contract
+ * @notice Contract writers can inherit this contract in order to create on-demand OCR requests
+ */
+abstract contract OCR2DRClient is OCR2DRClientInterface {
+    OCR2DROracleInterface private s_oracle;
```

Comment:
Maybe change this to `don`? ...hmm, maybe not

Reply:
Yeah I understand what you mean. This may change once a billing registry gets added in.